### PR TITLE
Fix sys.getsizeof

### PR DESCRIPTION
**Pull request by qingshi163**

### CODE REVIEW

File: Lib/test/test_ioctl.py [hunk 1]
```diff
@@ -66,8 +66,6 @@ def test_ioctl_mutate_2048(self):
         # Test with a larger buffer, just for the record.
         self._check_ioctl_mutate_len(2048)
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
```
Comment (DimitrisJim) [inline Lib/test/test_ioctl.py:70]: fyi, I think this test is flaky, it passes locally for me too but I don't think it passes on CI

File: vm/src/stdlib/sys.rs [hunk 2]
```diff
@@ -419,10 +419,30 @@ mod sys {
...
+        match sizeof() {
+            Ok(res) => Ok(res),
+            Err(err) => {
+                if let Some(default) = args.default {
+                    Ok(default)
+                } else {
+                    Err(err)
+                }
+            }
+        }
```
Comment (youknowone) [inline vm/src/stdlib/sys.rs:444]:
```suggestion
        sizeof().map_err(|err| args.default.ok_or(err))
```

**Review by youknowone — APPROVED** [review]

File: Lib/test/test_ioctl.py (hunk 1 above)
Comment (qingshi163) [reply Lib/test/test_ioctl.py:70]: surprise! it passed!!! I don't know why either

**Comment by youknowone** [comment]
👍

**Pull request merged**